### PR TITLE
Display flip properties

### LINKED ISSUE #336: Make flip-attributes dynamically show/hide-able
There are a lot of interesting things for every flip. Some users might not be interested in all or interested in others than normally displayed.
Develop a system that allows hiding properties and allow users to change the displayed properties via a settings view/page.
Requirements:
- all fields should be toggable 
- selection should be saved

### LINKED ISSUE #338: Display Flip properties
every flip now also have the array `props` containing a string array that is sorted by relevance and composed of relevant properties of the flip including:
- middas paid
- ulti enchants
- lvl 6+ enchants

This list should be displayed with a variable cut off (configurable by the user).
E.g. users wants to display a maximum of 2 of these properties if there are more they are hidden

### PR DESCRIPTION
**Pull request by matthias-luger**
fixes #338 
fixes #336

### CODE REVIEW

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-368.flou21.de

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-368.flou21.de

**Review by Ekwav — CHANGES_REQUESTED** [review]
I am missing any option to remove the horizontal lines.
There may now be two lines next to each other which make not a lot of sense

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-368.flou21.de

File: src/components/Flipper/FlipCustomize/FlipCustomize.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    settings = JSON.parse(getSetting(FLIP_CUSTOMIZING_KEY));
+} catch {
+    settings = {
+        hideCost: false,
+        hideEstimatedProfit: false,
+        hideLowestBin: false,
+        hideMedianPrice: false,
+        hideSeller: false,
+        hideVolume: false,
+        maxExtraInfoFields: 3
```
Comment (Ekwav) [inline src/components/Flipper/FlipCustomize/FlipCustomize.tsx:19]: this object is redundant

**Review by Ekwav — CHANGES_REQUESTED** [review]

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-368.flou21.de

**Review by Ekwav — APPROVED** [review]

**Pull request merged**